### ISSUE TITLE
Question About inference result

### ISSUE THREAD

**Issue by MaverickBeta** [ffhibnese/CGNC_Targeted_Adversarial_Attacks#5]
Dear authors,

Thank you for your excellent work—much appreciated! I have a small question regarding the use of inference.py. I am wondering if there are any specific steps I need to take before running eval.py after completing training for 10 epochs. I just want to make sure I am following the same procedure as you, since I am currently getting quite different results on my side. It is possible that the issue is on my end, but I would appreciate it if you could clarify the intended workflow.

Thank you very much for your help!

Best regards,
Hongyi

**Comment by ffhibnese** [comment]
Hi, Hongyi

Thanks for your interest in our work! There are no additional steps required between completing training for 10 epochs and running eval.py.

For the performance discrepancy, I suggest checking whether your training hyperparameters completely align with ours. 
Also, due to the instability of training a generator, there is a chance that the optimization process is stuck in a local minimum. You can repeat the experiment to verify this point.

Best wishes,
Hao Fang

**Comment by MaverickBeta** [comment]
May I ask what is the loss you get after 10 epoch training? I wonder if my training has some problems.

Best,
Hongyi

**Comment by ffhibnese** [comment]
> May I ask what is the loss you get after 10 epoch training? I wonder if my training has some problems.
> 
> Best, Hongyi

It's around 0.04.

**Comment by ffhibnese** [comment]
> May I ask what is the loss you get after 10 epoch training? I wonder if my training has some problems.
> 
> Best, Hongyi

It's around 0.04.

**Comment by MaverickBeta** [comment]
Hi Hao Fang,

I have been noticed that the loss I got is very close to what you get but it oscillated between a quite large scale from 0.085 to 0.012 during epoch 9(the last one). I wonder if this situation is normal or my training data isn't clean enough so that the loss decreased abnormally. 

Best,
Hongyi

**Comment by MaverickBeta** [comment]
Hi Hao Fang,

I have been struggling with reproduce the same performance as you mention in your paper. Is it possible to offer a checkpoint so that I could quickly validate my result.

Best,
Hongyi

**Comment by ffhibnese** [comment]
> Hi Hao Fang,
> 
> I have been struggling with reproduce the same performance as you mention in your paper. Is it possible to offer a checkpoint so that I could quickly validate my result.
> 
> Best, Hongyi

The pretrained generators have been provided. Please refer to the README.md file for more details.

**Comment by QiyuZhu1** [comment]
> Dear authors,
> 
> Thank you for your excellent work—much appreciated! I have a small question regarding the use of inference.py. I am wondering if there are any specific steps I need to take before running eval.py after completing training for 10 epochs. I just want to make sure I am following the same procedure as you, since I am currently getting quite different results on my side. It is possible that the issue is on my end, but I would appreciate it if you could clarify the intended workflow.
> 
> Thank you very much for your help!
> 
> Best regards, Hongyi
Hello, my situation is identical to yours. I was wondering whether the results of your subsequent training matched those mentioned in the article? If so, might I ask how you achieved this?
